engine/access/rpc/backend/backend_accounts.go

```diff
@@ -80,6 +80,126 @@ func (b *backendAccounts) GetAccountAtBlockHeight(
 	return account, nil
 }
 
+// GetAccountBalance returns the account balance at the latest sealed block.
+// Alias for GetAccountBalanceAtLatestBlock
+func (b *backendAccounts) GetAccountBalance(ctx context.Context, address flow.Address) (uint64, error) {
+	return b.GetAccountBalanceAtLatestBlock(ctx, address)
+}
+
+// GetAccountBalanceAtLatestBlock returns the account balance at the latest sealed block.
+func (b *backendAccounts) GetAccountBalanceAtLatestBlock(ctx context.Context, address flow.Address) (uint64, error) {
+	sealed, err := b.state.Sealed().Head()
+	if err != nil {
+		err := irrecoverable.NewExceptionf("failed to lookup sealed header: %w", err)
+		irrecoverable.Throw(ctx, err)
+		return 0, err
+	}
+
+	sealedBlockID := sealed.ID()
+
+	if b.scriptExecutor != nil {
+		accountBalance, err := b.scriptExecutor.GetAccountBalance(ctx, address, sealed.Height)
```

Comment:
to include the available balance, you'll need to expose `GetAccountAvailableBalance`, then call that as well.